config/routes.rb

```diff
@@ -1,6 +1,43 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+
+  scope module: 'api', path: "api" do 
+
+  scope :v1 do 
+    devise_for :users, path: '', path_names: {
+      sign_in: "login",
+      sign_out: "logout",
+      registration: "signup"
+      },
+      controllers: {
+        sessions: "api/v1/sessions",
+        registrations: "api/v1/registrations"
+      }
+    end
+  end
+
+  root 'homes#index'
```

Comment:
- I just see the Home controller in another folder, but it looks like there's no view for this. I invite you to add this, anyway is a bug, and it could be fixed. Thanks for reading me.